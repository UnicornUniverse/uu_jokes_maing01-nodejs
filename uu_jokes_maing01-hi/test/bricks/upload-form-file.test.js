import React from "react";
import { shallow } from "enzyme";
import UuJokes from "uu_jokes_main-client";

describe(`UuJokes.Bricks.UploadFormFile`, () => {
  it(`default props`, () => {
    const wrapper = shallow(<UuJokes.Bricks.UploadFormFile />);
    expect(wrapper).toMatchSnapshot();
  });
});
