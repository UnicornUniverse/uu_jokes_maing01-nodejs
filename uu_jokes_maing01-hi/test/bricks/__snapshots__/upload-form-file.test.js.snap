// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UuJokes.Bricks.UploadFormFile 1`] = `<formModal />`;
